src/main/java/Duke.java

```diff
@@ -1,10 +1,163 @@
+import java.util.Scanner;
+
 public class Duke {
+    public static int inputCount = 0;
+    public static Task[] tasks = new Task[100];
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        String input;
+        Scanner in = new Scanner(System.in);
+
+
+        System.out.println("Hello! I'm Duke\n" + "What can I do for you?\n");
+
+        input = in.nextLine();
+
+        while (!(input.equals("bye"))) {
+
+            if (input.equals("list")) {
+                input = printList(in);
+
+            } else if (input.contains("done")) {
+                input = testTaskDone(input, in);
+
+            } else if (input.contains("todo")) {
+                try {
+                    testInput(input); // test todo input
+
+                    Task whatToDo = toDoMethod(input);
+                    System.out.println("    Got it. I've added this task:\n" + "    " + whatToDo + "\n"
+                            + "    Now you have " + inputCount + " tasks in the list.");
+
+                    input = in.nextLine();
+                } catch (DukeExceptions e3) {
+                    System.out.println("☹ OOPS!!! The description of a todo cannot be empty.");
+                    input = in.nextLine();
+                }
+
+
+            } else if (input.contains("deadline")) {
+                try {
+                    Task whatDeadline = deadlineMethod(input);
+                    System.out.println("    Got it. I've added this task:\n" + "    " + whatDeadline + "\n"
+                            + "    Now you have " + inputCount + " tasks in the list.");
+
+                    input = in.nextLine();
+                } catch (StringIndexOutOfBoundsException e1) {
+                    System.out.println("input = " + input + "\n☹ OOPS!!! Please retype your input!");
+                    input = in.nextLine();
+                } catch (ArrayIndexOutOfBoundsException e2) {
+                    System.out.println("you have typed in = " + input + "\n☹ OOPS!!! where is your /by my friend? ");
+                    input = in.nextLine();
+                }
+
+            } else if (input.contains("event")) {
+                try {
+                    Task whatEvent = eventMethod(input);
+                    System.out.println("    Got it. I've added this task:\n" + "    " + whatEvent + "\n"
+                            + "    Now you have " + inputCount + " tasks in the list.");
+                    
+                    input = in.nextLine();
```

Comment:
It looks like `input = in.nextLine();` is called in many `else if` blocks, perhaps you could consider calling it only once to avoid repetitive code.